Package/Assets/GDK-Tools/Source/Editor/GdkEditorHelpers.cs

```diff
@@ -13,10 +13,23 @@ namespace Microsoft.GameCore.Tools
 {
     public class GdkEditorHelpers : ScriptableObject
     {
-        public static string GdkToolsPath { get; } = Path.Combine(RegUtil.GetRegKey(RegUtil.HKEY_LOCAL_MACHINE, @"SOFTWARE\WOW6432Node\Microsoft\GDK", "InstallPath"), "bin");
+        public static string GdkToolsPath 
```

Comment:
Actually thought about making this change anyway so that the path will still be retrieved if the GDK is installed after a project is opened